spec/shared/school_creator.rb

```diff
@@ -0,0 +1,76 @@
+def create_schools_at_status(preorder_status:, count: 1, responsible_body: nil)
```

Comment:
this works so we should leave for now, but we should somehow move this into the factories in the future so there is one single api to create these things